### ISSUE TITLE
Still having book download issues

### ISSUE THREAD

**Issue by sybergh0st** [rmcrackan/Libation#82]
I have never gotten your program to work for me, in any version.  That said, I know that at least a portion, however big or small, is on my end, as others seem to be able to get it to work.  I was able to, in version 5.3.6 get my US, and UK libraries to sync up, and it recognizes all of the titles, and their covers.  Once selected, every title gives me errors, and nothing downloads.  Yesterday, it had me upgrade to yesterday's latest version.  Same scenario.  Today, it had me upgrade to 5.4.9.  It gave me errors on the first couple of titles, and has been downloading some, ever since.  I am attaching yesterday's log, to this message, and will follow with today's when the downloading completes.
[_Log202108.log](https://github.com/rmcrackan/Libation/files/6925892/_Log202108.log)
[Log202108.log](https://github.com/rmcrackan/Libation/files/6925893/Log202108.log)

**Comment by Mbucari** [comment]
Audible changed their API and broke Libation 5.4.8 and earlier. Download v5.4.9 and delete everything in %temp%\Libation\DownloadsInProgress.

**Comment by sybergh0st** [comment]
I have done as you suggested.  Just an update to before I did many did succeed, but not all.  After I did the fresh install of 5.4.9, and cleared out the temp folder, I still have titles that give me the same error.  Maybe Libation cannot do these types of titles, as they are slightly different.  An example of one of the stubborn type is The Third Level, by Jack Finney.

**Comment by Mbucari** [comment]
I see you uploaded a log in which The Third Level failed. That log was made with Libation version 5.3.6.2. Can you please try again with the latest Libation release and submit that log instead?

**Comment by Mbucari** [comment]
> as they are slightly different.
Can you please elaborate?

**Comment by sybergh0st** [comment]
Elaborate: The title is a free "book" to anyone with an audible account.  I don't think it's one of the Pus books, but it functions similar. 

[_Log202108.log](https://github.com/rmcrackan/Libation/files/6926195/_Log202108.log)
[Log202108.log](https://github.com/rmcrackan/Libation/files/6926196/Log202108.log)

**Comment by Mbucari** [comment]
The Third Level (asin B09B58W92Z) is a podcast episode. Episodes aren't currently supported by Libation, but this particular book is not being correctly identified as an episode so it's not being filtered out properly.

**Comment by rmcrackan** [comment]
No reason we couldn’t identify episodes eventually. We’d just need to figure out the edge cases. Not much demand/incentive at the moment. Is there something you would like?

**Comment by sybergh0st** [comment]
Not particularly.  I just figured I would report the issue of Libation recognizing it, but that it would not download this type.  The other portion (the bigger portion) has been remedied, by using the method from issue 81.  It was in the closed issues section, and I didn't see it, and I'm new to the posting section of github, and wasn't aware to search the closed topics.  I was, however pointed to 81, after I posted, but before Mbucari responded.  The same fix worked for me, in that respect.  Also, thank both of you for the program, and the prompt responses.
